pro/src/pages/Accessibility/Accessibility.tsx

```diff
@@ -0,0 +1,56 @@
+import { AppLayout } from 'app/AppLayout'
+import strokeRigthIcon from 'icons/stroke-right.svg'
+import { ButtonLink } from 'ui-kit/Button/ButtonLink'
+import { ButtonVariant, IconPositionEnum } from 'ui-kit/Button/types'
+
+import styles from './Accessibility.module.scss'
+
+export const Accessibility = () => {
+  return (
+    <AppLayout>
+      <h1>Informations d’accessibilité</h1>
+
+      <div className={styles['menu-accessibility']}>
+        <ButtonLink
+          link={{
+            isExternal: false,
+            to: `accessibilite/engagements`,
+          }}
```

Comment:
`isExternal` n'est pas obligatoire. Pareil pour les autres.
```suggestion
          link={{
            to: `accessibilite/engagements`,
          }}
```